source/routing/isochrone.cpp

```diff
@@ -161,6 +161,39 @@ struct InfoCircle {
         duration_left(duration_left) {}
 };
 
+static bool within_info_circle(const InfoCircle& circle,
+                              const std::vector<InfoCircle>& multi_poly,
+                              const double& speed) {
+    if (multi_poly.empty()) {return false;}
+    auto begin = multi_poly.begin();
+    const auto end = multi_poly.end();
```

Comment:
That's better in generic programming, but in this context, I find `.begin()` and `.end()` more readable.